app/assets/stylesheets/views/_covid.scss

```diff
@@ -75,8 +79,12 @@ $covid-grey: #272828;
     margin-bottom: govuk-spacing(6);
   }
 
-  @include govuk-media-query($from: tablet) {
-    background-position: 0 3px;
+  // IE10/11 specific - vertical align in flexbox is ignored
+  // unless an explicit height is set
+  @include govuk-media-query($from: desktop) {
+    @media screen and (min-width:0\0) {
```

Comment:
`0\0`? - can this not just be 0?

Reply:
No - this is an IE10/11 specific hack. See the commit message for details.